### PR TITLE
added scss variables for palette colors, fonts and .button class

### PR DESCRIPTION
**Pull request by p-guelfi**
SCSS variables for color pallette, fonts, and .button class

### CODE REVIEW

File: app/assets/stylesheets/config/_fonts.scss [hunk 1]
```diff
@@ -1,16 +1,34 @@
 // Import Google fonts
-@import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700|Geologica:700,800|Khula:400&display=swap'); // Ensure Khula Regular is included
 
 // Define fonts for body and headers
 $body-font: "Work Sans", "Helvetica", "sans-serif";
```
Comment (DavidDicke) [inline app/assets/stylesheets/config/_fonts.scss:5]: Let's just change the body font to Khula, then we don't need the paragraph variable.

**Review by DavidDicke — COMMENTED** [review]
Looks great overall. Jut left one comment! Thanks Pablo. Ready to merge after the tiny change if you ask me.

File: app/assets/stylesheets/config/_fonts.scss (hunk 1 above)
Comment (p-guelfi) [reply app/assets/stylesheets/config/_fonts.scss]: Yes that makes sense!

**Comment by moelgazzar** [comment]
LGTM!

**Review by moelgazzar — APPROVED** [review]

**Review by moelgazzar — APPROVED** [review]

**Pull request merged**